docs/samples/drift-detection/alibi-detect/cifar10/trigger.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Trigger
+metadata:
+  name: vaeoutlier-trigger
+  namespace: default
+spec:
+  filter:
+    sourceAndType:
+      type: org.kubeflow.serving.inference.request
```

Comment:
Where is this defined?

Reply:
https://github.com/kubeflow/kfserving/blob/0e47113a058b0581e796fd890d8eae58556dde7b/pkg/logger/worker.go#L14-L15